src/main/java/com/igot/cb/masterdata/service/ValidationService.java

```diff
@@ -0,0 +1,49 @@
+package com.igot.cb.masterdata.service;
+
+import com.igot.cb.util.ApiResponse;
+import com.igot.cb.util.CbServerProperties;
+import com.igot.cb.util.ProjectUtil;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ValidationService {
```

Comment:
bring all the required validations into this class -- for degree and other objects